ci/publish.js

```diff
@@ -1,8 +1,12 @@
+import cp from "child_process";
 import fs from "fs";
 import path from "path";
-import cp from "child_process";
-import { promisify } from "util";
 import { fileURLToPath } from "url";
+import { promisify } from "util";
+
+import debug from "debug";
+
+const log = debug("waku:ci:publish");
```

Comment:
If we want to migrate build scripts to use ‘log’ library then we probably need to do the same to ‘deploy.js’
But I think better is to change rules of ‘eslint’.